pkg/cli/osm_controller_utils.go

```diff
@@ -0,0 +1,22 @@
+package cli
```

Comment:
This seems like a useful function.
I don't agree `package cli` is the right place.  If I needed a tool to list the pods for the OSM Controller service, I would not expect it to be in the `cli` package.

I'd recommend moving this function in `pkg/controller/pods.go`.